src/clincoded/static/components/variant_central/interpretation/segregation/evidenceSheet.js

```diff
@@ -144,6 +157,27 @@ let EvidenceSheet = createReactClass({
         }
     },
 
+    /**
+     * Check that hpoData and proband_hpo_ids are the same
+     * If not, returns array of rejected ids for form validation
+     * @param {array} inputIds
+     */
+
+    filterOutExtraHpo(inputIds) {
+        const hpoData = this.state.hpoData;
+        if (hpoData && hpoData.length) {
+            const idsFromHpoData = [];
+            hpoData.forEach(obj =>{
+                idsFromHpoData.push(obj.hpoId);
+            });
+            if (inputIds.length > idsFromHpoData.length) {
+                return inputIds.filter(id => !idsFromHpoData.includes(id));
```

Comment:
Need to make sure the inputIds list has unique ids.  If not, even the length is the same, the lists may not match.